x-pack/legacy/plugins/siem/public/components/ml/api/get_ml_capabilities.ts

```diff
@@ -22,18 +22,14 @@ export interface Body {
   maxExamples: number;
 }
 
-export const getMlCapabilities = async (
-  kbnVersion: string,
-  signal: AbortSignal
-): Promise<MlCapabilities> => {
+export const getMlCapabilities = async (signal: AbortSignal): Promise<MlCapabilities> => {
   const response = await fetch(`${chrome.getBasePath()}/api/ml/ml_capabilities`, {
     method: 'GET',
     credentials: 'same-origin',
     headers: {
-      'kbn-system-api': 'true',
       'content-Type': 'application/json',
```

Comment:
Another spot I named `content-Type` funny. Sorry again. Feel free to change that to all lower case `content-type` if you want (optional)

Reply:
ditto on these being gone in the next PR.